pennylane/qinfo/transforms.py

```diff
@@ -181,8 +181,8 @@ def _torch_jac(circ):
     def wrapper(*args, **kwargs):
         loss = functools.partial(circ, **kwargs)
         if len(args) > 1:
-            return torch.autograd.functional.jacobian(loss, (args))
-        return torch.autograd.functional.jacobian(loss, *args)
+            return torch.autograd.functional.jacobian(loss, args, create_graph=True)
```

Comment:
what does this do?

Reply:
The issue was that the value of the Jacobian wasn't being "watched", which caused the gradient computation later to be incorrect. Adding `create_graph=True` creates a path back to the input from the Jacobian, which fixes it.

Reply:
Ah okay so in torch I have to specify that I want the jacobian to be differentiable (watched)? Really strange that it computed something of the right shape then but with wrong values ...

Reply:
Right, the gradient was still computed because the contribution from the QNode probabilities was non-zero, but there was no contribution from the Jacobian term.